Skype/UCWA/MeTasks.md

```diff
@@ -1,4 +1,7 @@
-
+---
+title: me tasks for skype for business 2015
+description: "Me tasks refers to topics showing code samples of tasks that are related to the **me** user, the currently logged-on user." 
```

Comment:
I think we can leave out the aserisks around "me" here.

Reply:
removed.